t/151-intercept-log.t

```diff
@@ -0,0 +1,195 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_on();
+#workers(2);
+log_level('error');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 3 - 2);
+
+#no_diff();
+#no_long_string();
+run_tests();
+
+__DATA__
+
+=== TEST 1: sanity
+--- http_config
+    lua_intercept_log 4m;
+--- config
+    location /t {
+        access_by_lua_block {
+            ngx.log(ngx.ERR, "enter 1")
+            ngx.log(ngx.ERR, "enter 11")
+
+            local t = ngx.get_log()
+            ngx.say("log length:", #t)
+        }
+    }
+--- request
+GET /t
+--- response_body
+log length:341
+--- grep_error_log eval
+qr/enter \d+/
+--- grep_error_log_out eval
+[
+"enter 1
+enter 11
```

Comment:
The output misses many important info like log level, timestamps, connection ID, request ID, request context info. For example, a complete error log message should look like this:

```
2017/04/11 12:29:48 [error] 51725#0: *14 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /main HTTP/1.1", subrequest: "/memc", upstream: "memcached://127.0.0.1:11211", host: "localhost"
```